<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1526</width>
    <height>852</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>autoQuant</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0" columnstretch="0,0,0,0,0,0,0,0,0,0,0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="6" column="10">
       <widget class="QRadioButton" name="radioButton_4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="9" column="3">
       <widget class="QPlainTextEdit" name="plainTextEdit_5">
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;I&lt;span style=&quot; vertical-align:sub;&quot;&gt;0 avg&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>LED</string>
        </property>
       </widget>
      </item>
      <item row="9" column="8">
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="26" column="2">
       <widget class="QPushButton" name="plotQuantButton">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Calculate QY</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>ml</string>
        </property>
       </widget>
      </item>
      <item row="23" column="2">
       <widget class="QPushButton" name="LoadTrans">
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Load Reagent</string>
        </property>
       </widget>
      </item>
      <item row="6" column="6">
       <widget class="QPlainTextEdit" name="plainTextEdit_4">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;K &lt;span style=&quot; vertical-align:sub;&quot;&gt;A â†’ B &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="6" column="4" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>mW</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLabel" name="Header_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Power</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Volume</string>
        </property>
       </widget>
      </item>
      <item row="8" column="3" colspan="3">
       <widget class="QLabel" name="Header_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Wavelength of Irradiation</string>
        </property>
       </widget>
      </item>
      <item row="9" column="4" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_9">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>nm</string>
        </property>
       </widget>
      </item>
      <item row="23" column="3">
       <widget class="QPushButton" name="LoadCis">
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Load Product</string>
        </property>
       </widget>
      </item>
      <item row="6" column="11">
       <widget class="QPushButton" name="loadDataButton">
        <property name="maximumSize">
         <size>
          <width>125</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Load Data</string>
        </property>
       </widget>
      </item>
      <item row="9" column="9" colspan="2">
       <widget class="QPushButton" name="LoadLED">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Load LED Emission Data</string>
        </property>
       </widget>
      </item>
      <item row="7" column="9">
       <widget class="QPushButton" name="baselineCorrectionButton">
        <property name="maximumSize">
         <size>
          <width>125</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Baseline Correction</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="7" column="11">
       <widget class="QPushButton" name="calculatePowerButton">
        <property name="maximumSize">
         <size>
          <width>125</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Calculate Total Power</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPlainTextEdit" name="plainTextEdit_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="9" column="1" alignment="Qt::AlignRight">
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="4" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_6">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;s&lt;span style=&quot; vertical-align:super;&quot;&gt;-1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="24" column="2">
       <widget class="QPushButton" name="plotLEDButton">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Plot/Process LED</string>
        </property>
       </widget>
      </item>
      <item row="6" column="5">
       <widget class="QLabel" name="label_11">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; vertical-align:super;&quot;&gt;+&lt;/span&gt;/&lt;span style=&quot; vertical-align:sub;&quot;&gt;-&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" alignment="Qt::AlignRight">
       <widget class="QRadioButton" name="radioButton_3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="7" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>mW</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QPlainTextEdit" name="plainTextEdit_3">
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="3" colspan="3">
       <widget class="QLabel" name="Header_1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Experimental Parameters</string>
        </property>
       </widget>
      </item>
      <item row="23" column="5" colspan="3">
       <widget class="QPushButton" name="LoadSpectra">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Load Irradiation Spectra</string>
        </property>
       </widget>
      </item>
      <item row="24" column="5" colspan="2">
       <widget class="QPushButton" name="plotEpsilonButton">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Plot Epsilons</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1526</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
